### ISSUE TITLE
Publish on NPM

### ISSUE THREAD

**Issue by melanke** [jmaister/excellentexport#50]
Could you Publish this repo on NPM?

**Comment by jmaister** [comment]
Published on NPM!
+ excellentexport@1.5.0

**Comment by qinfanpeng** [comment]
It seems that we can use something like this:

```javascript
const ExcellentExport = require('excellentexport')
// Or
import ExcellentExport = from 'excellentexport'
```

So it would be better if it can support things like this

**Comment by jmaister** [comment]
I am figuring out how to make the compatible with bower and npm at the same time.

Any suggestions ?

**Comment by jmaister** [comment]
Fixed on version 2.0.0.
